.buildkite/pipeline.yml

```diff
@@ -89,47 +86,95 @@ steps:
 
   - label: ':npm: NPM Build (Staging)'
     command: 'buildkite-scripts/scripts/npm-build.sh'
-    branches: '!master'
+    branches: 'develop'
     env:
       REACT_APP_VIRTRU_ENV: 'staging'
     agents:
       - *agent_queue
 
+  - label: ':npm: NPM Build (Develop)'
+    command: 'buildkite-scripts/scripts/npm-build.sh'
+    branches: '!master !develop'
+    env:
+      REACT_APP_VIRTRU_ENV: 'develop'
+    agents:
+      - *agent_queue
+
   - wait
 
+  ####
+  # UNBLOCKED DEPLOYMENTS
+  ####
+
+  ####
+  # STAGING
+  # Deploy and Clear Cache
+  ####
```

Comment:
Nit but can the deploy and build steps appear in the same order with regard to the environment? Makes it easier to read, i.e. prod, staging, develop or the reverse